core/src/operations/mod.rs

```diff
@@ -3,6 +3,9 @@ use core::fmt;
 
 mod advice;
 pub use advice::AdviceInjector;
+mod debug;
+pub use debug::DebugOptions;
+mod tests;
```

Comment:
For modules which are used for test purposes only, we usually add `#[cfg(test)]` attribute.